sim/paladin/retribution/retribution_test.go

```diff
@@ -0,0 +1,226 @@
+package retribution
+
+import (
+	"testing"
+
+	_ "github.com/wowsims/sod/sim/common" // imported to get item effects included.
```

Comment:
Item effects have their own tests so I dont feel this is needed here

Reply:
Fixed in https://github.com/wowsims/sod/pull/302/commits/4b628b513d4a78739a4105be686f8a750f920c29